roxie/ccd/ccdlistener.cpp

```diff
@@ -1455,7 +1456,8 @@ class RoxieProtocolMsgContext : implements IHpccProtocolMsgContext, public CInte
 
         ensureContextLogger();
 
-        Owned<ISpan> requestSpan = queryTraceManager().createServerSpan("request", allHeaders, flags);
+        const char * spanName = !isEmptyString(queryName) ? queryName : "run_query";
+        Owned<ISpan> requestSpan = queryTraceManager().createServerSpan(spanName, allHeaders, flags);
```

Comment:
The name of the deployed roxie query is used as the span name.

Reply:
I do wonder if prefixing with the queryset name (generally in AKS this is just the roxie name) would help find queries from the same roxie.. but knowing that for sure will come down to experience.